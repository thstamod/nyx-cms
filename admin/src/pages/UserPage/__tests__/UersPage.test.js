import React from 'react';
import '@testing-library/jest-dom/extend-expect';
import { render } from '@testing-library/react';
import UsersPage from '../UserPage';

test('Users page Page renders', () => {
  const { asFragment } = render(<UsersPage />);

  expect(asFragment()).toMatchSnapshot();
});
