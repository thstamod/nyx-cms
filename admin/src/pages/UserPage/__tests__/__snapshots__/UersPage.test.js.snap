// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Users page Page renders 1`] = `
<DocumentFragment>
  <div
    class="generalContainer"
  >
    <h1>
      User page
    </h1>
  </div>
</DocumentFragment>
`;
